cadence/FUSD.cdc

```diff
@@ -0,0 +1,226 @@
+import FungibleToken from "./FungibleToken.cdc"
```

Comment:
The FungibleToken should already be deployed since it's used by FlowToken. Is there a good way to import the existing contract instead of deploying it again?